benchmark/fs/bench-stat.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+
+const bench = common.createBenchmark(main, {
+  n: [1e4],
+  kind: ['lstat', 'stat']
+});
+
+
+function main(conf) {
+  const n = conf.n >>> 0;
+  const fn = fs[conf.kind];
```

Comment:
I don't know how much it matters/affects things, but maybe we should change this variable name to avoid shadowing?